webpack/assets/javascripts/react_app/routes/Models/ModelsPage/components/ModelsPageContent.js

```diff
@@ -8,21 +8,14 @@ import ModelDeleteModal from './ModelDeleteModal';
 import LoadingPage from '../../../common/LoadingPage';
 import { withRenderHandler } from '../../../../common/HOC';
 
-const ModelsPageContent = ({
-  models,
-  search,
-  sort,
-  fetchAndPush,
-  itemCount,
-}) => {
+const ModelsPageContent = ({ models, sort, fetchAndPush, itemCount }) => {
   const [toDelete, setToDelete] = useState({});
 
   return (
     <React.Fragment>
       <ModelDeleteModal toDelete={toDelete} fetchAndPush={fetchAndPush} />
       <ModelsTable
         results={models}
-        search={search}
```

Comment:
looks like the search bar update the URL on search,
but the table content doesn't get updated

Reply:
After the useApi hook update I needed to use `useEffect` for `setAPIOptions` to update the table